package.xml

```diff
@@ -1,13 +1,22 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>moveit_opw_kinematics_plugin</name>
-  <version>0.0.0</version>
-  <description>MoveIt! KinematicsBase plugin for industrial robots.</description>
+  <version>0.2.0</version>
+  <description>
+    <p>MoveIt kinematics plugin for industrial robots.</p>
+    <p> This plugin uses an analytical inverse kinematic library, opw_kinematics, to calculate the inverse kinematics for industrial robots with 6 degrees of freedom, two parallel joints, and a spherical wrist.</p>
```

Comment:
not sure if there are any rules about line lengths but you could add a line break here somewhere